algorithms/exemplar/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "openie-algorithm-relationshipg3-api",
+  "version": "1.0.0",
+  "description": "Get the Music relationships by running the exemplar Algorithm on a set of sentences.",
+  "main": "index.js",
+  "scripts": {
+	"prestart": "npm install",
+	"start": "node index.js"
+  },
+  "keywords": [
+    "swagger"
+  ],
+  "license": "ISC",
+  "private": true,
+  "dependencies": {
+    "connect": "^3.2.0",
+    "js-yaml": "^3.3.0",
+    "swagger-tools": "0.10.1",
+    "clone": "^2.1.1",
+    "fs": "0.0.1-security"
```

Comment:
running `npm shrinkwrap` creates a `npm-shrinkwrap.json` file which you have to check in.

Also this line (L20) looks wrong. `fs` is a module included in node.js and doesn't need to be in the package.json